modules/product/src/Form/AddToCartForm.php

```diff
@@ -50,6 +55,18 @@ class AddToCartForm extends FormBase {
   protected $storeContext;
 
   /**
+   * The entity field manager.
+   *
+   * @var \Drupal\Core\Entity\EntityFieldManager
+   */
+  protected $entityFieldManager;
```

Comment:
The code ended up not using $entityFieldManager, so this needs to be removed (along with unused use statements, constructor changes).